forem/forem
# Transliterate article titles to slugs

## What type of PR is this? (check all applicable)


## Description

Article titles written in languages with non-Roman alphabets (Cyrillic, Greek, Hebrew, etc) have the [entire title stripped from the slug](https://forem.dev/varhal/comment/3ig).

## Related Tickets & Documents

- https://forem.dev/varhal/comment/3ig

## QA Instructions, Screenshots, Recordings

- Publish an article with a title that is completely in a non-Roman alphabet
  - [Example in Russian](https://github.com/forem/forem/pull/15051/files#diff-3c7d126e0813eeff65d4e818cf77bd10c432266976097a5a4efb853505d8bc57R546)
- The URL for the article should contain those characters transliterated to ASCII

### UI accessibility concerns?

Ideally, nothing should change on this front.

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [x] I will share this change in a [Changelog](https://forem.dev/t/changelog) or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_
